one/Cargo.toml

```diff
@@ -48,8 +48,11 @@ tracing.workspace = true
 tokio-util.workspace = true
 tokio-metrics = "0.3.1"
 tokio-prometheus-client = "0.1"
-
-
+clio = "0.3.4"
```

Comment:
do we still need this dependency? not sure about the others but I think we removed clio at least